Dockerfile

```diff
@@ -5,6 +5,22 @@ MAINTAINER Juan Marin Otero <juan.marin.otero@gmail.com>
 RUN yum -y install epel-release
 RUN yum -y install nginx
 
+# Install npm
+RUN yum install -y npm
```

Comment:
I'm guessing the version of `npm` available from CentOS 7 EPEL is pretty stale.  In [grasshopper-loader's `Dockerfile`](https://github.com/cfpb/grasshopper-loader/blob/master/Dockerfile#L11) @wpears is installing the latest version via _nodesource_.  Not sure if there are actual useful features in this version, but probably a good idea to get on the same version across projects.

The Joyent site also recommends [this approach](https://github.com/joyent/node/wiki/installing-node.js-via-package-manager#enterprise-linux-and-fedora).  They don't say it on these docs, but there is a `https://rpm.nodesource.com/setup_0.12` so we can specify the exact version like @wpears does for a Debian-based install.